backend/api/src/pin-comment.ts

```diff
@@ -0,0 +1,40 @@
+import * as admin from 'firebase-admin'
+import { Contract } from 'common/contract'
+import { isAdminId, isModId } from 'common/envs/constants'
+import { getUser, revalidateContractStaticProps } from 'shared/utils'
+import { APIError, type APIHandler } from './helpers/endpoint'
+
+export const pinComment: APIHandler<'pin-comment'> = async (
+  { commentPath },
```

Comment:
Hmm, I think this is a lot more confusing than just passing the `contractId` and `commentId` and reconstructing the path here.

Reply:
I was just doing the same method we currently use for hide-comment.ts

Still don't fully understand backend stuff so just trying to stick to our current code. Very happy for you to explain further though.